src/controller/CourseController.ts

```diff
@@ -60,6 +60,23 @@ export class CourseController {
         }
     }
 
+    async sort(request: Request, response: Response, next?: NextFunction) {
+        try {
+            let start: number = request.query["_start"];
+            let end: number = request.query["_end"];
+            const order = request.query["_order"];
+            const sort = request.query["_sort"];
+            return await this.courseRepository
+                .createQueryBuilder("course")
+                .orderBy(sort, order)
+                .limit(end - start)
+                .skip(start)
```

Comment:
ideally, i would like to use the `WHERE` clause instead of `skip` or `offset` because where is usually preferred when the table has a ton of records.

so i wanted to ask if we could have the `id` in the course table be a `long` vs a `uuid`  to allow the where clause 🤔

Reply:
this is no biggie though if we keep uuids 👍

Reply:
I think we chose `uuid` since we wanted the `ids` to be unique and autogenerated without doing too much work lolol. 
@emaela and @nicolesorial what do you think? Keep `uuid` or have the `id` be a `long`?

Reply:
Yea I've been wondering about the `uuid`s as well because they're difficult to work with -- I'm pretty sure we can still have the `id`s be unique to their respective tables automatically. I did some research and it looks like you don't _really_ need `uuid`s unless you're working with multiple databases where uniqueness may be more important (i.e. similar tables in different dbs). My only concern is that if we change this to an `id`, we should probably change all the identifiers to `id`s in all the tables for consistency? I'm here for that though, it's much easier visually